straxen/plugins/led_cal/led_calibration.py

```diff
@@ -65,6 +65,25 @@ class LEDCalibration(strax.Plugin):
         ),
     )
 
+    expected_led_position = straxen.URLConfig(
+        default=88,
+        infer_type=False,
+        help=(
+            "Expected LED position given the delayed set on the LED pulse. "
+            "This is used as default when no hits or less than a certain amount "
+            "are identified."
+        ),
+    )
+
+    minimum_hits_requirement = straxen.URLConfig(
+        default=500,
```

Comment:
For the bottom array I get around 300-500 hits per PMT and chunk. The noise run has O(1) hits. I think we should decrease the value to around 100.